### DISCUSSION TITLE
Parent is found in wrong package

### DISCUSSION THREAD

**Discussion by VerebLaszlo** [LemonAppDev/konsist#554] [Report a Problem]
### Tell us what version of Konsist you use.

0.12.1

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

Parent.kt
```
package org.myapp
interface Parent
```
Child.kt
```
package org.myapp.impl
sealed class Child : Parent
```
Test
`val parent = Konsist.scopeFromProduction().classes().filter{ it.name.equals("Child") }[0].parents[0]`

### Actual behaviour

It finds the Parent interface in the Child.kt file where it was referenced when defining the Child class.
`parent.containingFile` is Child.kt
`parent.packagee.text = "org.myapp.impl"`

### Expected behaviour

I think it should find in the Parent.kt file.
`parent.containingFile` should be Parent.kt
`parent.packagee.text = "org.myapp"`

**Comment by nataliapeterwas** [comment]
Currently this is an expected behaviour as we are just parsing stirngs (existing in the `Child.kt`) file.
We have a ticket in backlog to improve this.

**Comment by nataliapeterwas** [comment]
Will be fixed in next release 0.14.0 https://github.com/LemonAppDev/konsist/pull/709.
Thx for the report.

**Comment by nataliapeterwas** [comment]
A new version v0.14.0 has been released. It contains the fix.
See [v0.14.0 release](https://github.com/LemonAppDev/konsist/releases/tag/v0.14.0)
